modules/azure_k8s_base/azure-k8s-base.json

```diff
@@ -23,6 +23,21 @@
       "description": "Additional configuration for nginx ingress. [Available options](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#configuration-options)",
       "default": {}
     },
+    "cert_manager_values": {
+      "type": "object",
+      "description": "Certificate Manager helm chart additional values. [Available options](https://artifacthub.io/packages/helm/cert-manager/cert-manager?modal=values)",
+      "default": {}
+    },
+    "linkerd_values": {
```

Comment:
To go further, instead of hard coding annotation in the pods, we should use `podAnnotations` in our `/modules/aws_k8s_base/tf_module/values-ha.yaml`.

These annotations would be added in the values file and removed from the code.
```
        config.linkerd.io/skip-outbound-ports: "8126,5432,3306"
        config.linkerd.io/proxy-cpu-limit: "0.1"
        config.linkerd.io/proxy-cpu-request: "0.05"
        config.linkerd.io/proxy-memory-limit: "20Mi"
        config.linkerd.io/proxy-memory-request: "10Mi"
```

Reply:
These values are not just for HA linkerd, but for the default availability as well. 
Either we can make 2 values file, one with the high availability which we already have, and we can make one for the regular availability as well, so the users can then control them as well.